src/tournament-mode/remote-peer-menu.tsx

```diff
@@ -4,12 +4,20 @@ import {
   MenuItem,
   Icon,
   Collapse,
+  Button,
+  Position,
+  Toaster,
+  Intent,
   H3,
 } from "@blueprintjs/core";
 import { IconNames } from "@blueprintjs/icons";
 import { InputButtonPair } from "../controls/input-button-pair";
 import { displayFromPeerId, useRemotePeers } from "./remote-peers";
 
+const AppToaster = Toaster.create({
+  position: Position.BOTTOM
+});
```

Comment:
The app actually has a shared global toaster that can be imported instead of creating a new one here